lib/builder.js

```diff
@@ -157,6 +160,14 @@ Builder.prototype.build = function (willReadStringTree) {
               })
           })
         }).catch(function(e) {
+          // because `readAndReturnNodeFor` is recursive
+          // it does not make sense to throw multiple build errors,
+          // we can just report a failure once.
+          if (!isThrown) {
+            isThrown = true;
```

Comment:
Chatted with @twokul in slack, and he is going to update to ensure that this `isThrown` is reset when the next build starts.